### ISSUE TITLE
Auto-Complete when searching for users

### ISSUE THREAD

**Issue by ghost** [tieto/sipe#75]
It would be great if there could be auto-complete when searching for users in pidgin.

**Comment by xhaakon** [comment]
I've also noticed the query always returns only a single record. I search for 'Thomas' and get one name. We definitely have more of those.

**Comment by joakim-tjernlund** [comment]
Yes, please! I was just about to file an issue w.r.t both comments above, especially comment 2
I also want a shortcut so I don't have to click through Accounts -> actual account -> Contact Search
